src/components/DownloadLink/DownloadLink.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+import DownloadLink from './DownloadLink';
+
+describe('DownloadLink', () => {
+  it('renders correctly', () => {
+    render(<DownloadLink href="/file.pdf">Download</DownloadLink>);
+    const linkElement = screen.getByText('Download');
+    expect(linkElement).toBeInTheDocument();
+  });
```

Comment:
Wouldn't this test be redundant, since you need the component to render in order to run your tests below?